src/views/historical/hospitalization-hotspot/dashboard.vue

```diff
@@ -0,0 +1,504 @@
+        />
+        <router-link
+          :to="`/dataset/hospitalization-hotspot?town=${controls.town}&stat=${controls.focusStat.value}&cluster=${activeCluster.cluster_id}&zoom=${zoomed}&date=${currentDate}#resources`"
+        >
+          <i class="fa fa-arrow-circle-down fa-2x centered" />
+        </router-link>
+      </HiddenContent>
+    </template>
+  </DashboardCard>
+
+  <DashboardCard id="resources" width="full">
+    <template #title
+      >What can I do to lower the chances that someone in my community needs to
+      go to the hospital?</template
+    >
+    <template #content>
+      <div>
+        <div class="px-4 content">
+          <h5>Make sure people know where to go to get COVID-19 vaccines:</h5>
+          <ul>
+            <li>
+              Visit our pages about COVID-19
+              <router-link class="is-link" to="/dataset/vaccine-gap"
+                >Vaccines
+              </router-link>
+              and
+              <router-link class="is-link" to="/dataset/booster-gap"
+                >Boosters
+              </router-link>
+              to learn more about where there are gaps in our state.
+            </li>
+            <li>
+              You can also let people know
+              <ExternalLink href="https://covid.ri.gov/vaccination#athome">
+                how to get vaccinated at home </ExternalLink
+              >. This is a good option for people who may have a harder time
+              accessing a clinic.
+            </li>
+          </ul>
+          <h5>
+            Make sure people know where to go to get treatment for COVID-19:
+          </h5>
+          <ul>
+            <li>
+              Help people
+              <ExternalLink href="https://covid.ri.gov/detect/testing"
+                >get tested</ExternalLink
+              >
+              so they can start treatment as soon as possible.
+            </li>
+            <li>
+              Learn about
+              <ExternalLink
+                href="https://covid.ri.gov/treat/therapeutics?gclid=CjwKCAjwpayjBhAnEiwA-7ena53D2cKyXNp0MgIbBXPPz27355gBrR3UTplmWLarJHLRR3ZBFM_aJBoCMgwQAvD_BwE"
+              >
+                the treatments available
+              </ExternalLink>
+              for COVID-19.
+            </li>
+            <li>
+              Find out
+              <ExternalLink
+                href="https://covid-19-test-to-treat-locator-dhhs.hub.arcgis.com/"
+              >
+                where you can find treatment in your neighborhood </ExternalLink
+              >.
+            </li>
+          </ul>
+          <h5>
+            Make sure people know how to keep their health insurance coverage.
+          </h5>
+          <ul>
+            <li>
+              You can use this
+              <ExternalLink
+                href="https://staycovered.ri.gov/community-support/community-advocate-forum"
+                >community advocate toolkit</ExternalLink
+              >
```

Comment:
There must be something causing this? I also might be nitpicking a little too much 😔 

```suggestion
              <ExternalLink
                href="https://staycovered.ri.gov/community-support/community-advocate-forum"
              >
                community advocate toolkit
              </ExternalLink>
```